lib/sigsci-agent.sh

```diff
@@ -172,6 +172,7 @@ then
 
     cat > ${SIGSCI_CONFIG_FILE} <<EOT
 # Signal Sciences Reverse Proxy Config
+server-flavor = "sigsci-module-cloudfoundry"
```

Comment:
nit: The new config should go above the comment on the revproxy config (it is not the revproxy config)